bin/start-conjur.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -ex
+
+. ./utils.sh
+
+trap teardown EXIT
+
+# Type of Conjur to test against, 'all' or 'oss'
+export TEST_VERSION="${TEST_VERSION:-all}"
+announce "Compose Project Name: $COMPOSE_PROJECT_NAME
+     Conjur Test Version: $TEST_VERSION"
+
+main() {
+  # Build test container & start the cluster
+  announce "Pulling images..."
+  if oss_only; then
+      docker-compose -p $COMPOSE_PROJECT_NAME pull conjur
+  else
+      docker-compose -p $COMPOSE_PROJECT_NAME pull conjur cuke-master
+  fi
+  docker-compose -p $COMPOSE_PROJECT_NAME build postgres conjur cli5 cuke-master
+  echo "Done!"
+
+  announce "Starting Conjur environment..."
+  if oss_only; then
+    export CONJUR_DATA_KEY="$(docker-compose -p $COMPOSE_PROJECT_NAME run -T --no-deps conjur data-key generate)"
+    docker-compose -p $COMPOSE_PROJECT_NAME up --no-deps -d postgres conjur
+  else
+    export CONJUR_DATA_KEY="$(docker-compose -p $COMPOSE_PROJECT_NAME run -T --no-deps conjur data-key generate)"
+    docker-compose -p $COMPOSE_PROJECT_NAME up --no-deps -d postgres conjur cuke-master
+  fi
+  echo "Done!"
+
+  announce "Waiting for conjur to start..."
+  exec_on conjur conjurctl wait
+  if ! oss_only; then
+    exec_on cuke-master /opt/conjur/evoke/bin/wait_for_conjur
+  fi
+  echo "Done!"
+
+  api_key=$(exec_on conjur conjurctl role retrieve-key cucumber:user:admin | tr -d '\r')
+
+  if ! oss_only; then
+    announce "Running cuke setup..."
+    exec_on cuke-master bash -c 'conjur authn login -u admin -p secret'
+    exec_on cuke-master conjur user create --as-group security_admin alice
+    exec_on cuke-master conjur host create --as-group security_admin bob
+    exec_on cuke-master conjur variable create existent-variable-with-undefined-value
+
+    vars=(
```

Comment:
Maybe add a comment here explaining why these secrets are being added for `cuke-master`? I think they're being added as sample secrets that can be conveniently viewed in the dev environment?